src/lib/color-picker.service.ts

```diff
@@ -251,7 +251,7 @@ export class ColorPickerService {
         }
 
       default:
-        const allowHex8 = (alphaChannel === 'always' || alphaChannel === 'hex8');
```

Comment:
@sconix, I just noticed that the value `hex8` is not mentioned anywhere in the doc